src/CoolBoard.h

```diff
@@ -214,6 +216,13 @@ class CoolBoard {
    *register latch All I2C is over pins (2,14)
    */
   const int EnI2C = 5;
+
+  /**
+   *  Bootstrap pin,
+   *  double usage for flashing the Coolboard 
+   *  and start up the AP for further configuration
+   */
+  const int Bootstrap = 0;
```

Comment:
variables should be written in `camelCase` and not `PascalCase`, e.g. `bootstrap` and `enI2C` instead of `Bootstrap` and `EnI2C`

Reply:
There are some more of these PascalCase in the lib, take a look at CoolBoardSensors.h..
by the way, isn't CoolBoardSensors PascalCase too? 
maybe we should make a separate commit to clear the camel/PascalCase issue. at least I will create the issue to keep a trace of it...

Reply:
Yes, but `CoolBoardSensors` is a **class** name and as per conventions, `ClassNames` are `PascalCase`

Of course we should make separate commits to fix this kind of stuff in other places, but keep new code clear of those tiny compliance issues and rename the new `Bootstrap` variable to `bootstrap`